dashboard/app/models/experiments/experiment.rb

```diff
@@ -58,6 +58,18 @@ class Experiment < ApplicationRecord
       label: 'CSP Pilot',
       allow_joining_via_url: false
     },
+    # For facilitators in virtual academic year workshops for CSD
+    {
+      name: '20-21-virtual-AYW-CSD',
+      label: 'CSD Pilot',
+      allow_joining_via_url: true
+    },
+    # For facilitators in virtual academic year workshops for CSP
+    {
+      name: '20-21-virtual-AYW-CSP',
+      label: 'CSD Pilot',
```

Comment:
CSP Pilot?

Reply:
oops, didn't even notice these labels when I copy/pasted! thanks for catching.